app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_count = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            mask_count += 1
+    if mask_count > 0:
```

Comment:
Code Efficiency Issue: There's no need to compare `mask_count` to `0`. You can simply check `if mask_count:`. If `mask_count` is not `0`, it will be `True` in a boolean context.